.github/workflows/sbom-upload.yml

```diff
@@ -0,0 +1,14 @@
+name: SBOM upload
```

Comment:
just nitpicking, but wouldn't it be better when it is named `SBOM` instead of `SBOM upload` and the job `upload` instead of `SBOM-upload`? That way it would be shown as `SBOM/upload` instead of `SBOM upload/SBOM-upload`?

Reply:
Since this action is from Github marketplace, the preferences are to stick to name consistency.